### ISSUE TITLE
Babel reports errors in the Tamil and Chinese (China) translation files

### ISSUE THREAD

**Issue by mitya57** [sphinx-doc/sphinx#12624]
### Describe the bug

In Debian package, I am regenerating the compiled translation files from `.po` files during build.

And Babel reports 8 errors about the Tamil (`ta`) translation:
```
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:183: positional format placeholders are unbalanced
error:     in message string: 'புறக்கணித்தல் (தனிப்பட்ட கூறுகளை அமைக்க %r ஐப் பயன்படுத்தவும்) அகராதி கட்டமைப்பு அமைப்பை மேலெழுத முடியாது'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:190: positional format placeholders are unbalanced
error:     in message string: 'கட்டமைப்பு மதிப்புக்கு தவறான எண் %r, புறக்கணித்தல்'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:335: incompatible format for placeholder 1: 'r' and 's' are not compatible
error:     in message string: '" %s" என லெக்சிங் literal_block %r டோக்கனில் பிழை ஏற்பட்டது: %r. தளர்வான பயன்முறையில் மீண்டும் முயற்சிப்பது.'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:434: placeholders are incompatible
error:     in message string: 'கணக்கிடக்கூடிய முனை %ஏற்கனவே பதிவு செய்யப்பட்டுள்ளது'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:444: incompatible format for placeholder 1: 'r' and 's' are not compatible
error:     in message string: 'பதிப்பு %s முதல் %r நீட்டிப்பு ஏற்கனவே சூரரிமாச்சிலை உடன் இணைக்கப்பட்டுள்ளது; இந்த நீட்டிப்பு புறக்கணிக்கப்படுகிறது.'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:1996: positional format placeholders are unbalanced
error:     in message string: 'சேர்க்கப்பட்ட கோப்பு %r என்பது தவறானதாகத் தெரிகிறது, ஒரு: குறியாக்கம்: விருப்பம் கொடுக்க முயற்சிக்கவும்'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:2762: incompatible format for placeholder 1: 'r' and 's' are not compatible
error:     in message string: '%s இல் தவறான ரீசெக்ச் %r'
error: sphinx/locale/ta/LC_MESSAGES/sphinx.po:3778: placeholders are incompatible
error:     in message string: 'தீர்மானிக்க முடியாத மூல எழுத்துக்கள், "?"'
8 errors encountered in 'ta' locale, skipping
```

and one error about the Chinese (China) (`zh_CN`) translation:
```
error: sphinx/locale/zh_CN/LC_MESSAGES/sphinx.po:366: placeholders are incompatible
error:     in message string: '已略過無法讀取的文件'
1 errors encountered in 'zh_CN' locale, skipping
```

### How to Reproduce

1. Clone the Sphinx repository.
2. Run `python3 ./utils/babel_runner.py compile`.

### Environment Information

```text
Platform:              linux; (Linux-6.9.8-amd64-x86_64-with-glibc2.38)
Python version:        3.12.4 (main, Jun 12 2024, 19:06:53) [GCC 13.2.0])
Python implementation: CPython
Sphinx version:        7.4.7+/e439c6f33
Docutils version:      0.21.2
Jinja2 version:        3.1.3
Pygments version:      2.18.0
```

**Comment by AA-Turner** [comment]
I updated the translations workflow to avoid this, but it seems there is an error -- the .po and .js files were skipped, but the .mo files were updated.

See https://github.com/sphinx-doc/sphinx/pull/12563 -- the PR reports the failures in the cited languages.

A

**Comment by mitya57** [comment]
Ah, I did not know there is a separate project to report translation issues. Feel free to close this as duplicate.

> the .po and .js files were skipped, but the .mo files were updated

Correction: the **.mo** and .js files were skipped, but the **.po** files were updated.

**Comment by AA-Turner** [comment]
> Ah, I did not know there is a separate project to report translation issues.

That is only really meant for translations of Sphinx's documentation, but more translation-focussed discussion happens there.A

**Comment by AA-Turner** [comment]
Fixed in fa2ba7d5aeaff36aeb1a6c2f49846432fc932c7a and c6cd25f50dcc2a0bc369da80e75f105a4821d43b.

A
